docs/advanced-features/tabulator.md

```diff
@@ -0,0 +1,127 @@
+<!--pytest-codeblocks:skipfile-->
```

Comment:
any reason to have this?

Reply:
AFAIK it doesn't run SQL (also running testdocs with this line works for me)

Reply:
Okay, looks like it is safe to remove it.